### PR TITLE
Feature/search users

### PR DESCRIPTION
**Pull request by lenchv**

### CODE REVIEW

**Comment by ruig1992** [comment]
**TODO** :: Typeahead search and go to `/dashboard/messages/:user_id` when clicking in the typeahead search results.

File: resources/assets/js/features/chat/components/SearchUsers.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    }
+
+    render() {
+        const { translate } = this.props,
+            { preloader, options } = this.state,
+            icoClass = "search-users__icon fa" + (
+                preloader ? " fa-circle-o-notch fa-spin" : " fa-search"
+            );
+
+        /*<div className="input-group-addon search-users__icon-wrapper">
```
Comment (TatsuUkraine) [inline resources/assets/js/features/chat/components/SearchUsers.js]: no need commented code in new component

File: resources/assets/js/features/chat/components/SearchUsers.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    onSearchResultItemClick(userId) {
+        browserHistory.push(`/dashboard/messages/${userId}`);
+    }
+
+    renderMenuItemChildren(user, props, index) {
+        return (
+            <div key={user.id} onClick={() => this.onSearchResultItemClick(user.id)}>
+                <img
+                    src={getProfileAvatar(user.avatar)}
+                    style={{
```
Comment (TatsuUkraine) [inline resources/assets/js/features/chat/components/SearchUsers.js]: avoid using inline styles

**Review by TatsuUkraine — CHANGES_REQUESTED** [review]

**Comment by ruig1992** [comment]
@TatsuUkraine   This PR is not yet ready for review. But, thanks for your remarks! :-)

**Review by sergey-ben — APPROVED** [review]

**Review by TatsuUkraine — APPROVED** [review]

**Pull request merged**